.github/workflows/wasmcloud.yml

```diff
@@ -505,11 +505,31 @@ jobs:
       - uses: softprops/action-gh-release@v2
         if: startsWith(github.ref, 'refs/tags/wash-cli-v')
         with:
-          draft: true
+          draft: false
```

Comment:
note: I marked this as false because the homebrew/choco repos have to read the release